app/components/candidate_interface/course_choices_review_component.html.erb

```diff
@@ -45,6 +45,22 @@
           to see if the course will re-open or discuss alternatives
         </li>
       </ul>
+    <% elsif course_choice.chosen_study_mode_full? %>
```

Comment:
How about:
```suggestion
    <% elsif course_choice.course_option_full? && course_choice.other_study_modes_available? %>
```
with `other_study_modes_available?` focusing on the the second query only.

Maybe something similar for sites, too?